src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -549,7 +547,34 @@ private void SearchPackageInActivePackageSource(string searchText, bool useCache
                     ? Resx.Resources.Text_Loading
                     : string.Format(CultureInfo.CurrentCulture, Resx.Resources.Text_Searching, searchText);
 
-                _packageList.LoadItems(loader, loadingMessage, _uiLogger);
+                // start SearchAsync task for initial loading of packages
+                var searchResultTask = loader.SearchAsync(null, CancellationToken.None);
```

Comment:
does this still run a search even if UseCache is true? It's not clear why there is now an extra search when there wasn't before.